src/org/samples/basicapp/BasicApp.java

```diff
@@ -0,0 +1,60 @@
+package org.samples.basicapp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.moonlightcontroller.bal.BoxApplication;
+import org.moonlightcontroller.blocks.FromDevice;
+import org.moonlightcontroller.blocks.HeaderClassifier;
+import org.moonlightcontroller.blocks.ToDevice;
+import org.moonlightcontroller.processing.Connector;
+import org.openboxprotocol.protocol.HeaderField;
+import org.openboxprotocol.protocol.HeaderMatch;
+import org.openboxprotocol.protocol.IStatement;
+import org.openboxprotocol.protocol.OpenBoxHeaderMatch;
+import org.openboxprotocol.protocol.Priority;
+import org.openboxprotocol.protocol.Statement;
+import org.openboxprotocol.types.Port;
+
+public class BasicApp extends BoxApplication {
```

Comment:
Is this just some sample app?

It would be good, in general, to add some javadoc comment on each class with a short line about what its purpose is.